agent/consul/state/catalog_test.go

```diff
@@ -1937,7 +1937,31 @@ func TestStateStore_ServiceTagNodes_MultipleTags(t *testing.T) {
 		t.Fatalf("bad: %v", nodes)
 	}
 
-	idx, nodes, err = s.ServiceTagNodes(nil, "db", "dev")
+	// Test filtering on multiple tags
+	idx, nodes, err = s.ServiceTagNodes(nil, "db", []string{"v2", "slave"})
+	if err != nil {
+		t.Fatalf("err: %s", err)
```

Comment:
Same thing as before. I would use the require module here to get rid of the conditional blocks.